applications/connecting_applications_to_services/projecting-binding-data.adoc

```diff
@@ -0,0 +1,23 @@
+[id="projecting-binding-data"]
+= Projecting binding data
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: projecting-binding-data
+
+toc::[]
+
+This section provides information on how you can consume the binding data.
+
+== Consumption of binding data
+For accessing and consuming the binding data from a workload, this data has to be projected into the workload from a backing service. After the backing service exposes the binding data, {servicebinding-title} Operator automatically projects this set of data into the workload in the following methods:
```

Comment:
```suggestion
By default, the {servicebinding-title} Operator projects the binding data from the backing service into the workload by using files. To project the binding data by using environment variables, you configure the `.spec.bindAsFiles` parameter from the `ServiceBinding` resource.
```
This suggestion addresses several issues:

- The way it is used here, "from a workload" can have two different meanings: "For a workload to access and consume..." OR "To access and consume the binding data that comes from a workload..."
- "Automatically" and "manually" are weak words. In most cases, you can improve technical content by removing them and simply describing what the system or user does. 
- When possible, use the simple present tense and active voice instead of other verb forms, such as "projects" instead of "must be projected".
- Help the reader understand what their potential response to the content might be; describe system behavior and user actions separately and  address the reader directly as "you."

Reply:
Would it help to mention _why_ they might want to use environment variables. 
Is it safe to assume that the user already knows what "binding data" is?

Reply:
@rolfedh My response:

- I will change it to: "For a workload to access and consume..." 
- Here, the SBO feature automates the projection which is the highlight of the product and the projection feature. The reason why it is an important need/requirement for the user is explained in the Exposing binding data topic (for which I am yet to raise a PR) that will precede this topic. as such I am retaining this "automatically" word. 
- The 1st line is a requirement of the situation. The 2nd statement is the system behavior, not user action.

Reply:
> Would it help to mention _why_ they might want to use environment variables.
> Is it safe to assume that the user already knows what "binding data" is?

- Users for these procedures are typically Operator providers or users who create backing services. The PR for Exposing binding data will cover this in the Introduction. Given that, the user already is well aware of what binding data is. 

Using SBO, users can choose to project the data as either files (default method) or env variables. these are just options. No particular reason or benefit why a user must choose any one of them. By default, data is mounted by SBO as files.